src/components/Navigation.jsx

```diff
@@ -14,7 +14,7 @@ const Navigation = () => {
   return (
     <div className="fixed top-0 z-50">
       <div className="w-screen bg-witl-purple-300 flex items-center justify-between py-[1%] px-[2%] border-b-2 border-witl-white-100">
-        <Link className="flex flex-row space-x-4" href="/">
+        <Link className="flex flex-row space-x-4 p-2" href="/">
```

Comment:
```suggestion
        <Link className="flex flex-row space-x-4 py-2" href="/">
```
sry im being picky haha but make this change and ur good here